content/en/tracing/setup_overview/compatibility_requirements/python.md

```diff
@@ -34,13 +34,14 @@ The `ddtrace` library includes support for a number of web frameworks, including
 | [djangorestframework][6]  | >= 3.4            | https://ddtrace.readthedocs.io/en/stable/integrations.html#django  |
 | [Falcon][7]               | >= 1.0            | https://ddtrace.readthedocs.io/en/stable/integrations.html#falcon  |
 | [Flask][8]                | >= 0.10           | https://ddtrace.readthedocs.io/en/stable/integrations.html#flask   |
-| [Molten][9]               | >= 0.7.0          | https://ddtrace.readthedocs.io/en/stable/integrations.html#molten  |
-| [Pylons][10]              | >= 0.9.6          | https://ddtrace.readthedocs.io/en/stable/integrations.html#pylons  |
-| [Pyramid][11]             | >= 1.7            | https://ddtrace.readthedocs.io/en/stable/integrations.html#pyramid |
-| [pytest][12]              | >= 3.0            | https://ddtrace.readthedocs.io/en/stable/integrations.html#pytest  |
-| [Sanic][13]               | >= 19.6.0         | https://ddtrace.readthedocs.io/en/stable/integrations.html#sanic   |
-| [Starlette][14]           | >= 0.13.0         | https://ddtrace.readthedocs.io/en/stable/integrations.html#starlette |
-| [Tornado][15]             | >= 4.0            | https://ddtrace.readthedocs.io/en/stable/integrations.html#tornado |
+| [Fastapi][9]             | >= 0.51           | https://ddtrace.readthedocs.io/en/stable/integrations.html#fastapi |
```

Comment:
```suggestion
| [FastAPI][9]              | >= 0.51           | https://ddtrace.readthedocs.io/en/stable/integrations.html#fastapi |
```

Reply:
Based on their website, it looks like we should capitalize API in their brand name.